docs/index.html

```diff
@@ -7,5 +7,5 @@
 	</head>
 	<body>
 		<div id="app"></div>
-	<script type="text/javascript" src="build/bundle.f09e1b17.js"></script></body>
+	<script type="text/javascript" src="build/bundle.aa0f3cd7.js"></script></body>
```

Comment:
This is generated for documentation. Since we are using `codespec.github.io` domain for documentation, we need to commit the documentation files, too. Also, we need to generate every time when we change our components.